internal/ocm/ocm.go

```diff
@@ -383,3 +391,27 @@ func (csc *clusterServiceClient) ListBreakGlassCredentials(clusterInternalID Int
 	}
 	return BreakGlassCredentialListIterator{request: breakGlassCredentialsListRequest}
 }
+
+func (csc *clusterServiceClient) GetVersion(ctx context.Context, internalID InternalID) (*arohcpv1alpha1.Version, error) {
+	client, ok := internalID.GetHCPClusterVersionClient(csc.conn)
+	if !ok {
+		return nil, fmt.Errorf("OCM path is not a version: %s", internalID)
+	}
```

Comment:
I still could be proven wrong here, but I don't foresee a need to use `InternalID` with version resources.  Let's keep this simple for now and just take a version string argument, and get a version client similar to what you're doing in `ListVersions`.
```suggestion
func (csc *clusterServiceClient) GetVersion(ctx context.Context, id string) (*arohcpv1alpha1.Version, error) {
	client := csc.conn.AroHCP().V1alpha1().Versions().Version(id)
```